src/ruby-rails-postgres/.devcontainer/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN gem install rails webdrivers
 
 # Default value to allow debug server to serve content over GitHub Codespace's port forwarding service
 # The value is a comma-separated list of allowed domains 
-ENV RAILS_DEVELOPMENT_HOSTS=".githubpreview.dev,.app.github.dev"
+ENV RAILS_DEVELOPMENT_HOSTS=".githubpreview.dev,.preview.app.github.dev,.app.github.dev"
```

Comment:
Can you bump the patch version in here (ie. to `1.1.4`)? thanks

The Template will be republished only if the version doesn't exist.

https://github.com/devcontainers/templates/blob/1eb3ad739bf823ba5985cc73a06bee59e2b9bc22/src/ruby-rails-postgres/devcontainer-template.json#L3

Reply:
Ah good call!